product_docs/docs/biganimal/release/getting_started/creating_a_cluster/creating_a_dha_cluster.mdx

```diff
@@ -135,6 +135,10 @@ The following options aren't available when creating your cluster:
     
     See [Cluster networking architecture](01_cluster_networking) for more information. 
 
+1. Volume Snapshots
+
+Enable the **Volume Snapshots** to take the snapshot backups. The snapshot backups are stored on the disk in the same region without degrading the performance. It may increase the storage costs as per your cloud service providers. By default, the snapshot backups are stored on the disk for 30 days.
```

Comment:
The numbered list is broken because if the indentation 
```suggestion
   Enable the **Volume Snapshots** to take the snapshot backups. The snapshot backups are stored on the disk in the same region without degrading the performance. It may increase the storage costs as per your cloud service providers. By default, the snapshot backups are stored on the disk for 30 days.
```